pkg/parquet/row_reader.go

```diff
@@ -100,16 +103,16 @@ func NewBufferedRowReaderIterator(reader parquet.RowReader, bufferSize int) *Buf
 }
 
 func (r *BufferedRowReaderIterator) Next() bool {
-	if len(r.buff) > 1 {
-		r.buff = r.buff[1:]
+	if len(r.bufferedRows) > 1 {
+		r.bufferedRows = r.bufferedRows[1:]
```

Comment:
That old line surely reduced the capacity by`-1`


It is only when you cut of the end that the capacity stays the same, so `buf[:cap(buff)-1]`, will still have the same capactity.